backend/component/ghost/config.go

```diff
@@ -13,6 +13,38 @@ import (
 	"github.com/bytebase/bytebase/backend/store"
 )
 
+var defaultConfig = struct {
```

Comment:
const

Reply:
compiler complains that this is not a constant.